jest.config.js

```diff
@@ -10,6 +10,6 @@ module.exports = {
   ],
   transform: {
     '^.+\\.[t|j]sx?$': 'babel-jest',
-    '^.+\\.scss$': 'sass-jest',
+    '^.+\\.scss$': '<rootDir>/node_modules/jest-css-modules-transform',
```

Comment:
why did you change this transform?

Reply:
I was asked to install `jest-CSS-modules-transform` module in order to run  jest test